### PR TITLE
Migrate AssetHelperTest to Kotlin

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
Migrate AssetHelperTest to Kotlin

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt [hunk 1]
```diff
@@ -13,20 +13,18 @@
...
+import org.hamcrest.MatcherAssert
+import org.junit.Test
 
+class AssetHelperTest {
     @Test
-    public void guessMimeTypeTest() {
-        assertThat(AssetHelper.guessMimeType("test.txt"), is("text/plain"));
-        assertThat(AssetHelper.guessMimeType("test.png"), is("image/png"));
-        assertThat(AssetHelper.guessMimeType("test.zip"), is("application/zip"));
+    fun guessMimeTypeTest() {
```
Comment (mikehardy) [inline AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt:25]: Can you add this and re-push the second commit?

```suggestion
    @KotlinCleanup("is --> equalTo")
    fun guessMimeTypeTest() {
```

**Review by mikehardy — CHANGES_REQUESTED** [review]
Looks great! One small change to note the needed future change to a non reserved word

File: AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt (hunk 1 above)
Comment (oyeraghib) [reply AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt:26]: Okay, but what is it for ?

File: AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt [hunk 2]
```diff
@@ -13,20 +13,19 @@
...
 
+class AssetHelperTest {
     @Test
-    public void guessMimeTypeTest() {
-        assertThat(AssetHelper.guessMimeType("test.txt"), is("text/plain"));
-        assertThat(AssetHelper.guessMimeType("test.png"), is("image/png"));
-        assertThat(AssetHelper.guessMimeType("test.zip"), is("application/zip"));
+    @KotlinCleanup("is --> equalTo")
+    fun guessMimeTypeTest() {
+        MatcherAssert.assertThat(guessMimeType("test.txt"), CoreMatchers.`is`("text/plain"))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt:27]: Re-import `MatcherAssert` and `CoreMatchers`.

Remove the import of `AssetHelper` so it's `AssetHelper.guessMimeType`

The Kotlin Migration process often fails to handle imports, OR adds imports to static methods unnecessarily.

**Review by david-allison — CHANGES_REQUESTED** [review]
One nit regarding imports

File: AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt (hunk 2 above)
Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt]: Still an issue: 

> Remove the import of AssetHelper so it's AssetHelper.guessMimeType

**Review by david-allison — APPROVED** [review]
Awesome, thanks!

**Review by mikehardy — APPROVED** [review]
nice! Thank you

**Pull request merged**